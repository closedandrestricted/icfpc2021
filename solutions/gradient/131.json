{"vertices":[[20,115],[11,119],[16,121],[14,127],[135,178],[143,176],[46,143],[146,179],[155,181],[85,70],[86,82],[51,117],[95,72],[58,148],[99,79],[191,190],[63,185],[151,131],[97,163],[91,120],[91,31],[126,45],[167,154],[100,113],[143,30],[135,139],[211,102],[131,10],[140,88],[131,87],[155,103],[128,181],[128,118],[162,107],[151,96],[168,107],[148,88],[195,19],[146,54],[155,88],[103,97],[143,97],[158,83],[173,59],[151,139],[152,86],[150,99],[174,98],[154,76]]}