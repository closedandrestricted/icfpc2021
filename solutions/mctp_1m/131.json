{"vertices":[[4,56],[1,47],[3,49],[3,55],[2,58],[5,66],[15,18],[2,47],[8,53],[0,66],[3,78],[37,72],[0,57],[30,9],[1,66],[0,87],[56,80],[3,89],[3,80],[65,105],[11,21],[40,62],[25,53],[2,65],[23,8],[4,39],[1,35],[0,71],[36,62],[37,58],[32,85],[77,155],[49,102],[35,69],[33,78],[36,64],[35,67],[0,45],[38,58],[31,87],[41,80],[42,73],[34,93],[34,65],[65,115],[31,87],[35,75],[37,59],[7,45]]}