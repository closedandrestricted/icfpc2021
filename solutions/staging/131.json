{"vertices":[[21,115],[12,119],[18,122],[14,127],[135,178],[144,174],[48,144],[146,179],[155,181],[85,70],[86,82],[51,117],[95,72],[58,148],[99,79],[191,190],[63,185],[152,129],[99,164],[91,120],[91,31],[126,45],[167,154],[108,119],[142,29],[132,138],[211,102],[131,10],[133,95],[132,89],[155,102],[128,181],[128,118],[165,105],[151,96],[168,107],[148,88],[195,22],[150,52],[155,88],[100,97],[143,97],[156,82],[175,63],[151,139],[152,86],[150,99],[174,98],[150,84]]}