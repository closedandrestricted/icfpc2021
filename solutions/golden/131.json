{"vertices":[[36,138],[31,130],[43,136],[39,131],[124,162],[123,172],[81,150],[125,173],[134,176],[105,109],[101,99],[57,90],[111,117],[98,124],[109,108],[170,185],[179,115],[77,188],[104,86],[91,120],[89,44],[83,82],[155,149],[103,65],[153,62],[119,38],[115,133],[126,84],[141,94],[135,84],[157,82],[154,42],[135,99],[143,75],[157,75],[148,93],[171,69],[138,13],[174,98],[164,70],[123,95],[179,62],[168,65],[173,39],[192,12],[174,62],[187,63],[195,41],[168,60]]}